src/main/java/org/testng/TestRunner.java

```diff
@@ -1483,7 +1461,7 @@ public static int getVerbose() {
     return m_verbose;
   }
 
-  public void setVerbose(int n) {
+  public static void setVerbose(int n) {
```

Comment:
As each <test> node may have its own verbose value, `m_verbose` has to be non-static.